docs/kubernetes/AWSIntegration.md

```diff
@@ -0,0 +1,68 @@
+# Integration with an AWS Kubernetes cluster
+This guide will walk you through how to use Calico Networking with an existing AWS-Kubernetes cluster.
+
+## Requirements
+* A working Kubernetes Deployment on AWS
+    - While any AWS deployment will do, we recommend the Kubernetes [`kube-up` guide](https://github.com/kubernetes/kubernetes/blob/release-1.0/docs/getting-started-guides/aws.md) for AWS. This guide was created with the `kube-up` script in mind.
+* SSH access to your master and minions
+    - Unless otherwise specified, the `kube-up` script will create a `kube_aws_rsa` private key in the `~/.ssh` folder which you can use to access your AWS Instances.
+    - SSH in with the following command `ssh -i </path/to/key> ubuntu@<PUBLIC_IP>`
+
+## Preparing your master services
+### Setting up Calico's etcd backend
+On your master, download our [etcd manifest](https://raw.githubusercontent.com/projectcalico/calico-kubernetes-coreos-demo/ah3-config-update/manifests/calico-etcd.manifest) and replace all instances of `<PRIVATE_IPV4>` with your master's IP or hostname. Then, install the manifest in the `/etc/kubernetes/manifests/` directory. The kubelet on your master should automatically spin up a docker container for the new etcd which can be accessed on port 6666 of your master.
+
+### Reconfiguring the Kubernetes API and services
+For now, our plugin does not support token authentication for API access. While we implement more advanced security features, you will have to configure your `kube-apiserver` on the master to use an `insecure-bind-address` and an `insecure-port`. 
+* The `kube-up` script will implement a secure apiserver by default. To change this you will need to access the `/etc/kubernetes/manifests/kube-apiserver.manifest` file. In the `ExecStart` section, add `--insecure-bind-address=<PRIVATE_IPV4>` and `--insecure-port=8080`.
+* You may also have to reconfigure the `kube-controller-manager` and `kube-scheduler` manifests to point to the insecure private IP instead of the secure `127.0.0.1` loopback address.
```

Comment:
As talked about, this isn't clear on whether I should or should not reconfigure the manifests.  Are my manifests ok if I don't? I don't know.